charts/flyte/values-gcp.yaml

```diff
@@ -1,3 +1,186 @@
+userSettings:
+  googleProjectId: <GOOGLE_PROJECT_ID>
+  dbHost: <DB_HOST>
+  dbPassword: <DB_PASSWORD>
+  bucketName: <BUCKET_NAME>
+
+#
+# FLYTEADMIN
+#
+
+flyteadmin:
+  replicaCount: 2
+  serviceAccount:
+    # -- If the service account is created by you, make this false, else a new service account will be created and the iam-role-flyte will be added
+    # you can change the name of this role
+    create: true
```

Comment:
It creates separate KSAs for flyteadmin/flytepropeller/datacatalog but they are all linked to just a single GSA called `flyte` via workload identity for simplicity. It might make sense to make this more fine grained for better security.

Reply:
> Thanks for updating this! Looks great. There are a few `TODOs` in the index.rst. Are those meant to be filled in before merge?

@nicklofaso Yes I'd like to have complete docs for a minimal working setup on GCE. That's why it is still a draft.